README.md

```diff
@@ -4,8 +4,23 @@
 
 ### Setup
 
+First, you'll need to install:
+
+- `docker-compose`
+- `rustup`
+
+Then, for development you'll probably want to install:
+
 ```
 rustup component add rustfmt-preview clippy-preview
 ```
 
 If you're using VSCode and have the RLS extension installed (you should), it'll ask to install more components, just say yes.
+
+### Running
+
+In the repo root:
+
+```
+docker-compose up
+```
```

Comment:
I didn't actually add the docker-compose files in this PR, those are coming in the next one. I accidentally included this change tho, whoops.